libraries/matchers.rb

```diff
@@ -198,7 +198,7 @@ def delete_aws_route53_record(resource_name)
 
   # System Manager Parameter Store
   def get_aws_ssm_parameter_store(resource_name)
-    ChefSpec::Matchers::ResourceMatcher.new(:aws_parameter_store, :create, resource_name)
+    ChefSpec::Matchers::ResourceMatcher.new(:aws_parameter_store, :get, resource_name)
```

Comment:
Just a future note. This whole file can go away. ChefSpec auto generates matchers now so there's no need to write them out by hand.